src/components/pages/Users/AddUserPanel/AddUserPanel.tsx

```diff
@@ -57,12 +57,13 @@ const AddUserPanel = ({ close, setPanelWidth }: Props) => {
           ? `Unable to create local user: ${postIdentitiesError.response?.data.message}`
           : null
       }
-      onSubmit={async (
-        { email, firstName, lastName },
-        addGroups,
-        addRoles,
-        addEntitlements,
-      ) => {
+      onSubmit={async (values, addGroups, addRoles, addEntitlements) => {
+        if (!values) {
+          // Code execution won't arrive here, as the form can't be submitted
+          // without the email field populated, thus, values won't be null.
+          return;
+        }
```

Comment:
I think it could be good to pass an `isDirty` param, like this: https://github.com/canonical/rebac-admin/pull/180/files#diff-183fe252c20fd38d66e7edb9ed373d77f49788307239c7376f2c522d7cb8b328R44 which means you don't have to have this condition.